phys/module_sf_noahdrv.F

```diff
@@ -907,7 +1022,7 @@ SUBROUTINE lsm(DZ8W,QV3D,P8W3D,T3D,TSK,                      &
 !Fei: urban. for urban surface, if calling UCM, redefine the natural surface in cities as
 ! the "NATURAL" category in the VEGPARM.TBL
 	
-	   IF(SF_URBAN_PHYSICS == 1.OR. SF_URBAN_PHYSICS==2.OR.SF_URBAN_PHYSICS==3 ) THEN
+	   IF(SF_URBAN_PHYSICS == 1.OR. SF_URBAN_PHYSICS==2.OR.SF_URBAN_PHYSICS==3 .OR. SF_URBAN_PHYSICS==4) THEN  ! TEB opt 4
```

Comment:
@dmey 
By using the named parameters (such as NOAUCMSCHEME) instead of integers (such as 1), it clarifies things for the user, and you do not need to add decorating text ("  ! TEB opt 4").

Reply:
same as in https://github.com/wrf-model/WRF/pull/1052#discussion_r366447050.